src/components/ThumbnailImage.js

```diff
@@ -40,7 +42,7 @@ class ThumbnailImage extends PureComponent {
         const {width, height} = this.calculateThumbnailImageSize(props.imageWidth, props.imageHeight);
         this.state = {
             thumbnailWidth: width,
-            thumbnailHeight: height,
+            thumbnailHeight: Math.max(40, height),
```

Comment:
```suggestion
            thumbnailHeight,
```

Reply:
Code is not DRY moved logic to Math.max(40, height) to `calculateThumbnailImageSize` method

Reply:
Added.